frontend/src/components/payment/cards/CardInfo.module.scss

```diff
@@ -0,0 +1,41 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1fr, 4);
+  grid-template-rows: repeat(1fr, 4);
+
+  background-color: var(--bg-color-secondary);
+  border-radius: var(--border-radius);
+
+  padding: 10px;
```

Comment:
Just FYI, I usually try to do multiples of 4px

Reply:
Ah good to know! Perhaps we should define some common padding variables? :)

Reply:
Yeah, would be a good thing